Unity-Technologies/ScriptableRenderPipeline
# Universal/bugfixes

### Purpose of this PR
 - Fixed issue that caused objects to to render in scene view when Wireframe mode was selected. This was caused by post-processing. Now, post-processing is not executed if wireframe mode is selected.
 - Fixed issue that caused shaders containing HDRP string to be stripped from build.
 - Fixed Renderer Feature UI tooltips.

### QA
 Issue to validate fix:
   - Fogbugz 1191756 [not public]
   - Fogbugz 1175098 [not public]

---
### Testing status

**Manual Tests**: What did you do?
- [x ] Opened test project + Run graphic tests locally
- [x ] C# and shader warnings (supress shader cache to see them)
- [x ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.
